### ISSUE TITLE
Won't compile on r19

### ISSUE THREAD

**Issue by RamXX** [basho/basho_stats#8]
Due to the changes in r19, particularly the deprecation of the `random` module, basho_stats won't compile. This is affecting other projects that depend on basho_stats, like Udon.
```
==> basho_stats (compile)
Compiling /Users/rs/workspace/Erlang/udon/deps/basho_stats/src/basho_stats_rv.erl failed:
/Users/rs/workspace/Erlang/udon/deps/basho_stats/src/basho_stats_rv.erl:38: random:uniform/0: the 'random' module is deprecated; use the 'rand' module instead
/Users/rs/workspace/Erlang/udon/deps/basho_stats/src/basho_stats_rv.erl:44: random:uniform/0: the 'random' module is deprecated; use the 'rand' module instead
/Users/rs/workspace/Erlang/udon/deps/basho_stats/src/basho_stats_rv.erl:57: random:uniform/0: the 'random' module is deprecated; use the 'rand' module instead
/Users/rs/workspace/Erlang/udon/deps/basho_stats/src/basho_stats_rv.erl:58: random:uniform/0: the 'random' module is deprecated; use the 'rand' module instead
/Users/rs/workspace/Erlang/udon/deps/basho_stats/src/basho_stats_rv.erl:68: random:uniform/0: the 'random' module is deprecated; use the 'rand' module instead
ERROR: compile failed while processing /Users/rs/workspace/Erlang/udon/deps/basho_stats: rebar_abort
make: *** [compile] Error 1
```

**Comment by kesslerm** [comment]
Proposed fixes in #10

**Comment by bsparrow435** [comment]
Fixed. Thank you!!
